src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java

```diff
@@ -260,10 +260,22 @@ public void testGenotypesOnly(File input, File expected, List<String> extraArgs,
     public void assertMatchingGenotypesFromGenomicsDB(File input, File expected, Locatable interval, String reference) throws IOException {
         final File tempGenomicsDB = GenomicsDBTestUtils.createTempGenomicsDB(input, interval);
         final String genomicsDBUri = GenomicsDBTestUtils.makeGenomicsDBUri(tempGenomicsDB);
-        runGenotypeGVCFSAndAssertSomething(genomicsDBUri, expected, NO_EXTRA_ARGS, VariantContextTestUtils::assertVariantContextsHaveSameGenotypes, reference);
+        final List<String> args = new ArrayList<String>();
+        args.add("--"+GenotypeCalculationArgumentCollection.MAX_ALTERNATE_ALLELES_LONG_NAME);
+        args.add("2"); // Too small max_alternate_alleles arg to GenomicsDB, should fail
+        try {
+            File output = runGenotypeGVCFS(genomicsDBUri, expected, args, reference);
+            Assert.assertTrue(output.exists());
```

Comment:
If we're counting on this to fail instead of `assertTrue(output.exists())` we should  probably have a `Assert.fail("Should have thrown an exception")`